Bottom: 0b0cf95a3193ab6a1aed18d7c4cc6a1f1768ebf4
Top:    00842c93a8488fc7f3a8ad793f88ae8f65d63fe9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 12:42:14 +0000

Refresh of add-missing-return-type-to-0

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 2c75fa4..c5b2d71 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -517,6 +517,7 @@ next_cvc_from_other_poly (CVCList *start)
   return l;
 }
 
+static void
 cvc_list_dump (CVCList *list)
 {
   VNODE *node;
